target/linux/stm32/stm32mp1/config-6.6

```diff
@@ -274,7 +274,7 @@ CONFIG_KERNEL_GZIP=y
 CONFIG_KMAP_LOCAL=y
 CONFIG_KMAP_LOCAL_NON_LINEAR_PTE_ARRAY=y
 # CONFIG_LEDS_BRIGHTNESS_HW_CHANGED is not set
-# CONFIG_LEDS_CLASS_MULTICOLOR is not set
+# CONFIG_LEDS_CLASS_MULTICOLOR=Y
```

Comment:
Just remove this line. If there is no special setting in this file it will take the value from the generic configuration.